app/src/androidTest/java/org/mozilla/reference/browser/FirefoxTestApplication.kt

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser
+
+class FirefoxTestApplication
```

Comment:
Nit: should this extend `FirefoxApplication`?

Reply:
BrowserApplication on RB. Amended in the new pull request. Using this design required me to make the BrowserApplication class open.